README.md

```diff
@@ -47,7 +47,7 @@ WOFF is supported in IE 9+, Chrome, Firefox, iOS 5+, Android 4.4+.
 
 ## Advanced usage<a name="advanced"></a>
 
-### Loading all the web fonts
+### Loading all Origami provided web fonts
```

Comment:
My brain is struggling to wrap around this sentence, could we say something like 'Loading all web fonts provided by Origami' instead? 
There are a few spots in the README and the docs where the word order throws me off a little